core/src/mindustry/logic/LStatements.java

```diff
@@ -312,6 +312,49 @@ public LCategory category(){
         }
     }
 
+    @RegisterStatement("printchar")
+    public static class PrintCharStatement extends LStatement{
+        public String value = "65";
+
+        @Override
+        public void build(Table table){
+            table.add(" char ");
+            TextField field = field(table, value, str -> value = str).get();
+            table.button(b -> {
+                b.image(Icon.pencilSmall);
+                b.clicked(() -> showSelectTable(b, (t, hide) -> {
+                    t.row();
+                    t.table( i -> {
+                        i.left();
+                        int c = 0;
+                        for(char j=32;j<127;j++){
```

Comment:
no spaces around  ` = ` or after ; - look at other for-loops in the codebase and copy their style